lib/pavilion/config.py

```diff
@@ -472,13 +477,6 @@ class PavilionConfigLoader(yc.YamlConfigLoader):
             choices=['debug', 'info', 'warning', 'error', 'critical'],
             help_text="The minimum log level for messages sent to the pavilion "
                       "logfile."),
-        ExPathElem(
-            "result_log",
-            # Derive the default from the working directory, if a value isn't
-            # given.
-            help_text="Results are put in both the general log and a specific "
-                      "results log. This defaults to 'results.log' in the default "
-                      "working directory."),
```

Comment:
You should have Pavilion print a deprecation warning and configure a common logger for this.